ArchiSteamFarm/NLog/Logging.cs

```diff
@@ -235,6 +241,16 @@ internal static class Logging {
 		ArchiKestrel.OnNewHistoryTarget(historyTarget);
 	}
 
+	internal static async Task<string[]?> ReadLogFileLines() {
+		try {
+			return await File.ReadAllLinesAsync(SharedInfo.LogFile).ConfigureAwait(false);
```

Comment:
We can use something like `LogManager.Configuration.AllTargets.OfType<FileTarget>().FirstOrDefault()?.FileName` to get the actual filename in case user supplied custom `NLog.config`.

Reply:
I don't like it, you don't know what exists in that file. User might've declared two files, one for errors and one for info, and you'll get first one. I prefer to tell caller "sorry the file that should be used for that doesn't exist" rather than "I don't know what that is but here you go".

Sure, the same argument can be used about user modifying the default log.txt, but that we can't do anything about, and the call itself is *virtually correct*, as we return the correct file, just altered intentionally by the user, rather than "who knows what".

Reply:
> User might've declared two files

I thought of that too and would've loved an optional parameter for it. Since there is a `Name` property on the ~~`FileTarget`~~ `Target` class, this would've been easy to implement, giving the user the option to look at all their log-files in their UI and not just one specific file.

Reply:
That's something which makes sense to me, send PR :sunglasses: